### DISCUSSION TITLE
#11. VATZ biweekly meeting at `2022-06-23 23:00:00 CT` | `2022-06-24 13:00:00 KST`

### DISCUSSION THREAD

**Discussion by xellos00** [dsrvlabs/vatz#145] [General]
Retrospect: Sprint 10 

![image](https://user-images.githubusercontent.com/6308023/174168749-2eee2366-802f-45d9-b426-045e02f79917.png)
> 4/14

- Finished
   - #124 
   - #133 
   - #98 
   - #101
 
 - Push to Sprint 11
    - #69 
    - #85 
    - #87 
    - #96 
    - #118 
    - #119 
    - #120 
    - #121 
    - #122 
    - #127 

---

# What's been voted and confirmed. 

![image](https://user-images.githubusercontent.com/6308023/174162858-ff429adc-99b9-400c-910b-b6203dcaf068.png)

**Comment by xellos00** [comment]
# 동요

- #142
- #143
- #150 
- #151   
- #149

**Comment by rootwarp** [comment]
# 준교

- https://github.com/dsrvlabs/vatz/issues/133
- https://github.com/dsrvlabs/vatz/pull/154/files
- https://github.com/dsrvlabs/vatz-plugin-matic/issues/4

**Comment by meetrick** [comment]
# 황제
- https://github.com/dsrvlabs/vatz/issues/121
  - https://github.com/dsrvlabs/vatz-plugin-relayer
- https://github.com/dsrvlabs/vatz/issues/152
  - https://github.com/dsrvlabs/vatz-plugin-osmosis

**Comment by kim201212** [comment]
# 형기
- https://github.com/dsrvlabs/vatz/issues/156
   - https://github.com/dsrvlabs/vatz-plugin-persistence
- https://github.com/dsrvlabs/vatz/pull/155

**Comment by heejin-github** [comment]
## 희진
- #118 
  - https://github.com/dsrvlabs/vatz-plugin-solana/pull/8
  - https://github.com/dsrvlabs/vatz-plugin-solana/pull/9
  - https://github.com/dsrvlabs/vatz-plugin-solana/pull/10

**Comment by gnongs** [comment]
# 진홍
- #127
- #146
